data_specs/Sentinel-2_Level-2A_specs.ipynb

```diff
@@ -355,7 +355,48 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "**Table 5: AWS file path convention**\n",
+    "### Amazon Simple Notification Service "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Sentinel-2 Level-2A stacs can be received thanks to the Public Dataset Program."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Table 5: AWS SNS subscription details**"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "|AWS S3 details | |\n",
+    "|----------|-------------|\n",
+    "|SNS ARN | `arn:aws:sns:af-south-1:543785577597:deafrica-sentinel-2-scene-topic`|\n",
+    "| Region |`af-south-1` |\n",
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The SNS endpoints are in the AWS region `af-south-1` (Cape Town). Additional region specifications can be applied as follows:\n",
+    "\n",
+    "`aws sns list-topics --region=af-south-1 `\n",
```

Comment:
You really can't list these public topics like this... this is not helpful.

Reply:
removed